### ISSUE TITLE
Version number is incorrect in source code

### ISSUE THREAD

**Issue by dsharrison** [appiphony/appiphony-lightning-js#85]
When viewing the compiled source for `3.1.0` (such as `jquery.aljs-all-with-moment.js`) the version number in the header is incorrectly listed as `2.0.0`:

>  /* ------------------------------------------------------------
> Appiphony Lightning JS Beta
> 
> Version: 2.0.0
> Website: http://aljs.appiphony.com
> GitHub: https://github.com/appiphony/appiphony-lightning-js
> License: BSD 2-Clause License
> ------------------------------------------------------------ */

This caused some confusion for us and is likely to cause confusion for others as we are checking whether or not we are on the latest version. To fix this either the `jquery.aljs-init.js` file should have its header updated on each release or the header should be built using a gulp task where it can pull from the `package.json` (which has the correct version number).

**Comment by kevinberonilla** [comment]
You are correct, this was incorrectly implemented in the previous releases. In our upcoming Summer-supported release (4.0.0), version numbers will be updated and applied to each file.
